_data/projects/search/releases/7.1/7.1.0.Alpha1.yml

```
@@ -0,0 +1,27 @@
+date: 2023-09-05
+announcement_url: https://in.relation.to/2023/11/30/hibernate-search-7-1-0-Alpha1
+summary: >-
+  Introduction of vector search support in the Lucene backend,
```

Careful about the word "support"! ;D

```suggestion
  Vector search in the Lucene backend,
```